src/views/SignIn/index.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import { Alert, StatusBar } from 'react-native';
+
+import {
+    Title,
+    Subtitle,
+    Background,
+    FromContainer,
+    RegisterTitle,
+    RegisterButton,
+    ImageBackground,
+    RegisterContainer,
+    RegisterButtonText,
+    ContainerButton
+} from './styles';
+
+import bg from '@assets/bgLogin.png';
```

Comment:
Dá uma olhada nesse link para conseguir importar imagens: https://stackoverflow.com/questions/43823289/how-to-import-image-svg-png-in-a-react-component